docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -398,3 +398,13 @@ FAQs
         curl -LO "https://dl.k8s.io/release/v1.28.11/bin/linux/amd64/kubectl" && \
         sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl && \
         echo 'export PATH="$PATH:$HOME/.local/bin"' >> ~/.bashrc
+
+Multi-node jobs
+----------------
+
+:ref:`Multi-node jobs <dist-jobs>` are supported on Kubernetes. When a multi-node job is launched, each node in a SkyPilot cluster is provisioned as a separate pod.
+
+SkyPilot will attempt to place each pod on a different node in the cluster.
+
+SkyPilot will try to schedule all pods on a given cluster. If SkyPilot cannot schedule all pods on a given cluster (i.e. some or all of the pods cannot be scheduled),
+SkyPilot will fail over to another cluster or cloud.
```

Comment:
```suggestion
If you have multiple Kubernetes clusters, SkyPilot will schedule all the job pods in the same cluster. If SkyPilot cannot schedule all pods on a given Kubernetes cluster (i.e. some or all of the pods cannot be scheduled due to insufficient resources),
it will fail over to another Kubernetes cluster or cloud.
```